### PR TITLE
feat(core): put invitation status

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

resolves LOG-7975. provide management api to accept or revoke an organization invitation. since this pull is already large, the following change will be separate pulls:

- request error i18n
- update on magic links

## Testing

added a lot of api tests

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [x] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/core/src/routes/organization/invitations.ts [hunk 1]
```diff
@@ -64,6 +68,50 @@ export default function organizationInvitationRoutes<T extends AuthedRouter>(
...
+          acceptedUserId: true,
+        })
+        .extend({
+          status: z.enum([
+            OrganizationInvitationStatus.Accepted,
+            OrganizationInvitationStatus.Revoked,
+          ]),
+        }),
+      response: organizationInvitationEntityGuard,
+      status: [200],
```
Comment (simeng-li) [inline packages/core/src/routes/organization/invitations.ts:92]: exception status? 422, 404 e.g.

File: packages/core/src/routes/organization/invitations.openapi.json [hunk 2]
```diff
@@ -92,23 +81,41 @@
...
+                  "acceptedUserId": {
+                    "description": "The ID of the user who accepted the organization invitation. Required if the status is \"Accepted\"."
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
```
Comment (simeng-li) [inline packages/core/src/routes/organization/invitations.openapi.json:114]: ditto: exception status

File: packages/core/src/routes/organization/invitations.ts (hunk 1 above)
Comment (gao-sun) [reply packages/core/src/routes/organization/invitations.ts:92]: this is interesting. 422 can be successfully returned (see tests) while we don't have it here.

Comment (gao-sun) [reply packages/core/src/routes/organization/invitations.ts:92]: ah because we don't catch errors in `koaGuard`. i'll update this in a separate pull

**Review by simeng-li — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

**Pull request merged**